bootstrap/kubeadm/api/v1beta1/kubeadmconfig_types.go

```diff
@@ -675,19 +675,42 @@ type DiskSetup struct {
 	Filesystems []Filesystem `json:"filesystems,omitempty"`
 }
 
+// DiskLayout represents an array of partition specifications
+type DiskLayout []PartitionSpec
```

Comment:
I would avoid a type alias to a slice, it makes it harder to understand the API when looking at the go types (it looks like a struct for example if I look at the `Partition` usage)

Reply:
I will convert it to a struct

Reply:
I just meant you could inline this slice, rather than using an alias, I don't think it needs to be a struct does it?